data/human/transport missions.txt

```diff
@@ -3304,3 +3304,199 @@ mission "Spacediving amateurs"
 	on complete
 		payment 30000
 		dialog "As you descend through the upper atmosphere, you inform your skydivers that it's time to jump. However, after being able to see the distances and atmospheric conditions involved, all of them are far less keen on the idea than when you first picked them up. You bring them down to the landing pad, and they pay you a meager <payment> before hopping off."
+
+
+mission "Mafia Extortion"
+	minor
+	source
+		government "Pirate"
+	on accept
+		payment -200000
+		fail
+	to offer
+		random < 2
+		credits >= 2000000
+	on offer
+		conversation
+			`A man in a tailored suit, sunglasses, shiny shoes, and green hair approaches you. "Hey, that's a very nice ship you have there. Would be a shame if anything happened to it."`
+			`	You notice that he is missing a finger on each hand.`
+			choice
+				`	"Thank you. My name is <first> <last>. To whom do I have the pleasure of talking to?"`
```

Comment:
```suggestion
				`	"Thank you. My name is <first> <last>. To whom do I have the pleasure of speaking?`
```

(Otherwise the verb has two instances of "to" that apply to it)